document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/mapping/smallfull/mappers/SmallFullIXBRLMapperDecorator.java

```diff
@@ -535,6 +545,56 @@ private TangibleAssetsNetBookValue mapTangibleAssetsNetBookValue(TangibleApi tan
         return netBookValue;
     }
 
+    private LoansToDirectors mapLoansToDirectors(SmallFullApiData smallFull, Map<String, Integer> directorIndexes) {
```

Comment:
Just so I understand. This is mapping each loan to a director, yes?
I only ask because we've got an object called LoansToDirectors, but we're not mapping that to anything, I think that this method is mapping each loan to its director - is that right?
If so, maybe this method could be called mapEachLoanToItsDirector?

Reply:
This method maps loans to directors API data to its ixbrl format counterpart. The method has been named in a way such that it is consistent with the other methods, but can be changed if you have a strong preference?

Reply:
OK, leave as is.